buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -1,8 +1,8 @@
 object Versions {
     const val APP_COMPAT = "1.1.0"
-    const val KOTLIN = "1.5.10"
+    const val KOTLIN = "1.8.0"
```

Comment:
why not kotlin 1.9?

Reply:
Because all the versions are outdated by so many versions, so I've tried to make as small changes as possible. but https://github.com/androidbroadcast/ViewBindingPropertyDelegate this library requires minimum Kotlin 1.8.0. That's why I've migrated to this version. I'll try to bump it to 1.9.0 directly. If it will not require any more changes, so I can do it in one step